libs.springframework/external/spring-framework-4.0.1.RELEASE-notice.txt

```diff
@@ -0,0 +1,11 @@
+Spring Framework 4.0.1.RELEASE
+Copyright (c) 2002-2014 Pivotal, Inc.
+
+This product is licensed to you under the Apache License, Version 2.0
+(the "License"). You may not use this product except in compliance with
+the License.
+
+This product may include a number of subcomponents with separate
```

Comment:
This blurb seems pointless to add in the NOTICE file. The whole point of the NOTICE (and LICENSE) file is to be exhaustive, so there is nothing that 'may' be included...

Reply:
Agreed. I did this in the past and just overlooked, that the second and third paragraph are superfluous. This just leaves the copyright info.